### PR TITLE
readme of frontend update

### PR DESCRIPTION
**Pull request by Castella1313**
#67

I've update readme of frontend

### CODE REVIEW

**Comment by Castella1313** [comment]
@codewithvk I've made a few changes in readme of frontend, I'm not sure if I've done the right changes!

File: frontend/README.md [hunk 1]
```diff
@@ -1,70 +1,31 @@
...
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
+## 4.installing create-react-app
```
Comment (codewithvk) [inline frontend/README.md:11]: not need to re-install create-react-app

**Review by codewithvk — COMMENTED** [review]
@Castella1313 You can take reference from [here](https://github.com/HITK-TECH-Community/Community-Website).

Please do fast :)

File: frontend/README.md (hunk 1 above)
Comment (codewithvk) [reply frontend/README.md:11]: Basically, we want to installation step instead of the un-necessary thing.

**Comment by Castella1313** [comment]
@codewithvk I've made the changes accordingly ! Do have a look

**Pull request merged**